gulpfile.js

```diff
@@ -10,12 +10,16 @@ sass.compiler = require("node-sass");
 /* eslint-enable @typescript-eslint/no-var-requires */
 
 const scssFiles = ["**/*.scss", "!example/*.scss"];
+function throwSassError(e) {
+    sass.logError(e);
+    throw new Error("sass compilation failed", e);
```

Comment:
We could probably drop the e here, as it is logged much more readable in the line above. For the furture, we throw an error here to fail the build process if compilation fails. sass.logError only logs, it does not stop the build.